spec/spec_helper_local.rb

```diff
@@ -1,4 +1,8 @@
 require 'rbvmomi'
```

Comment:
This file is labeled `spec_helper_local` but I _think_ convention is that it should be `spec_helper_acceptance_local` unless these helpers are also used for unit tests?

Reply:
You're right, i'll rename it 👍

Reply:
this is still containing a lot of duplicated code to spec_helper_acceptance_local, please sort this out.